TicTacToeTests/GameTests.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  GameTests.swift
+//  TicTacToeTests
+//
+//  Created by Cora Jacobson on 9/23/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+@testable import TicTacToe
+
+class GameTests: XCTestCase {
+
+    func testRestart() {
+        var game = Game(board: GameBoard(), gameIsOver: false)
+        game.gameState = .active(.x)
+        
+        for x in 0..<3 {
+            for y in 0..<3 {
+                XCTAssertNil(game.board[(x, y)])
+            }
+        }
+        XCTAssertEqual(game.gameState, .active(.x))
+    }
```

Comment:
Since this test is called "testRestart", I would expect to see you call game.restart() here.